### PR TITLE
 Adding an image modal to the schedule image

### PR DESCRIPTION
**Pull request by jilaga**
I created an image modal as a solution to #471

### CODE REVIEW

File: js/scripts.js [hunk 1]
```diff
@@ -49,4 +49,111 @@ document.addEventListener('DOMContentLoaded', function() {
 			e.currentTarget.parentElement.parentElement.querySelector(".expertise-detail").classList.toggle('is-hidden');
 		});
 	}
-});
\ No newline at end of file
+});
+document.addEventListener('DOMContentLoaded', () => {
```
Comment (vasconsaurus) [inline js/scripts.js:53]: @jilaga if you don't mind me asking if this is a modal for our event schedule image, why are getting the navbar elements? This seems to repeat line 4 of the original file

**Comment by vasconsaurus** [comment]
Hi @jilaga! Could you maybe add some images of the new proposed behavior?

File: js/scripts.js (hunk 1 above)
Comment (jilaga) [reply js/scripts.js:53]: @vasconsaurus yes, I created a modal for the image so the contents of the schedule can be viewed full screen. I am a bit confused, Do you think it's an error? I would appreciate your assistance here.

**Comment by jilaga** [comment]
> Hi @jilaga! Could you maybe add some images of the new proposed behavior?

The cursor changes to a pointer on the image and opens full screen when clicked like this :

open modal

File: js/scripts.js (hunk 1 above)
Comment (vasconsaurus) [reply js/scripts.js:53]: At first glance, I think it's a bit odd, but I'm not sure. I'll copy your code over here and play with it a bit, if you don't mind 🙂 But I think I'll only be able to do that tomorrow

Comment (jilaga) [reply js/scripts.js:53]: > At first glance, I think it's a bit odd, but I'm not sure. I'll copy your code over here and play with it a bit, if you don't mind 🙂 But I think I'll only be able to do that tomorrow

Sure...go ahead 😄

**Comment by bebatut** [comment]
I am not sure having an image modal is the best solution there. It would be useful if we have a lot of image, but it is not really the case here. And the modal will be still limited by the window size.
The quick solution would be have a link to the image that will open in a new tab

**Comment by jilaga** [comment]
> I am not sure having an image modal is the best solution there. It would be useful if we have a lot of image, but it is not really the case here. And the modal will be still limited by the window size.
> 
> The quick solution would be have a link to the image that will open in a new tab

Okay,I understand better now. Let me work on it, Thank you.

**Pull request closed without merging**